### PR TITLE
order users alphabetically on the org manage_accounts form

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: templates/triggers/trigger_list.haml [hunk 1]
```diff
@@ -134,7 +134,7 @@
                           {{ obj.groups.all|join:", " }}
 
                         -if obj.schedule
-                          -if obj.schedule.next_fire
+                          -if obj.schedule.next_fire and not obj.is_archived
```
Comment (norkans7) [inline templates/triggers/trigger_list.haml]: Show "not scheduled" in archived list for schedule triggers

File: temba/orgs/models.py [hunk 2]
```diff
@@ -742,7 +742,7 @@ def get_org_surveyors(self):
 
     def get_org_users(self):
         org_users = self.get_org_admins() | self.get_org_editors() | self.get_org_viewers() | self.get_org_surveyors()
-        return org_users.distinct()
+        return org_users.distinct().order_by('email')
```
Comment (norkans7) [inline temba/orgs/models.py]: This will list accounts in manage accounts alphabetically

Comment (nicpottier) [inline temba/orgs/models.py]: Do invitations show up alphabetically as well? These are separate lists ya?

File: templates/triggers/trigger_list.haml (hunk 1 above)
Comment (nicpottier) [inline templates/triggers/trigger_list.haml]: Excellent, thanks for adding that.

**Comment by nicpottier** [comment]
Need to add 1 line of coverage. :)

File: temba/orgs/models.py (hunk 2 above)
Comment (nicpottier) [inline temba/orgs/models.py]: Where do pending invitations land? Below or in the same list?

Just don't think those should be intermixed.. We should have one alphabetical list for current users and below that pending invites in alphabetical order.

Comment (norkans7) [inline temba/orgs/models.py]: they are below not mixed

File: templates/orgs/org_manage_accounts.haml [hunk 3]
```diff
@@ -52,11 +52,13 @@
...
+          %td.permission
             .glyph.level-checkbox{class:'{% if invite.user_group == "S"%}checked{%endif%}'}
-          %td.check.permission
-            .hidden-input.hide
-              %input{type: 'checkbox'}
-            .glyph.level-checkbox
+          -#%td.check.permission
+          -#  .hidden-input.hide
+          -#    %input{type: 'checkbox'}
+          -#  .glyph.level-checkbox
```
Comment (nicpottier) [inline templates/orgs/org_manage_accounts.haml]: always remove, don't comment out

**Pull request merged**